src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -78,11 +78,27 @@
      */
     private Map<String, DetectorData> detectors;
 
+    public Map<String, DetectorData> getDetectors(){
+        return this.detectors;
+    }
+
     /**
      * The time since the last update call.
```

Comment:
could you change these two single line comments

Reply:
I would recommend you delete this comment altogether - every developer can infer that `timeSinceLastUpdate` means `The time since last update call`. This is as redundant as it gets.